### ISSUE TITLE
Ventilation/air filtering?

### ISSUE THREAD

**Issue by astearns** [w3c/TPAC2022-Health-rules#1]
I don’t see at a quick glance if there is anything the venue has done recently to improve ventilation and air filtering for the conference rooms and other areas in the complex. I assume this may factor in for some people’s calculus on whether they can physically attend.

**Comment by frivoal** [comment]
I strongly agree. In particular, I would like to see more resources targeted at ventilation and prevention of airborne transmission than at surface cleaning.

While there was a lot of emphasis early in the pandemic about washing hands and cleaning surfaces, there has since been a lot of scientific research showing that transmission of COVID-19 by surfaces was negligible, and that transmission was primary airborne.

Washing hands is good hygiene in general, and there's nothing wrong with providing hand sanitizers, but I have also heard that we were considering for example having a team of people who would come into the rooms at each breaks to clean the surfaces. I think this sort of things would be over-investing into protecting against the wrong threat model, and we should be paying more attention to opening windows, limiting re-circulation of unfiltered air, bringing in fresh air and/or filtering it, etc…

Some references highlighting the importance of airborne transmission as the relevant threat model (could dig out *many* more, that's just a small sample):
* https://www.nature.com/articles/d41586-021-00251-4
* https://www.annualreviews.org/doi/abs/10.1146/annurev-chembioeng-092220-111631
* https://www.sciencedirect.com/science/article/pii/S030438942103020X

See also the <abbr title="World Health Organization">WHO</abbr>'s  guidelines [to improve and ensure good indoor ventilation in the context of COVID-19](https://www.who.int/publications/i/item/9789240021280)

**Comment by svgeesus** [comment]
> While there was a lot of emphasis early in the pandemic about washing hands and cleaning surfaces, there has since been a lot of scientific research showing that transmission of COVID-19 by surfaces was negligible, and that transmission was primary airborne.

Exactly. And yet the rules seem aimed at minimizing droplet/surface transmission, ignoring airborne. Let's stick with the science, please.

**Comment by alexandralacourba** [comment]
The reason why we did not mention specific details about these aspects yet is because we are still working with the hotel on these aspects to make sure we have a good level of ventilation.

**Comment by swesthaver** [comment]
The Sheraton Wall Centre, Vancouver reached out to BC CDC for guidance on their air filtration system and upgraded their system. They upgraded their filters to MERV13 for HVAC and MUA units in common areas and meeting spaces.

**Comment by gosko** [comment]
Maybe it would be worthwhile to have a co2 air quality monitor on site? I don't know a lot about these but I have heard good things about the Aranet4 which costs ~$200. We wouldn't need one for each room, but if we had one or two on site we could periodically survey the rooms to understand whether the existing ventilation is adequate or needs adjustment. Knowing the co2 levels are low might help reduce anxiety for meeting participants, and knowing if co2 levels are high in a given space might be important info for those who are immunocompromised.

**Comment by fantasai** [comment]
@swesthaver I think an important question to ask is what percentage of their air intake is outside air, and is it possible to increase that to 100%. This will help both with COVID and with making sure there's enough oxygen in the meeting rooms to think intelligently. ;)

**Comment by swesthaver** [comment]
Hi Fantasai, I've asked the hotel your question. I expect to meet with my contact next week. 
Susan

**Comment by swesthaver** [comment]
Hi Fantasai,

My hotel contact said fans could be on at 100% for external, filtered air, but we would forfeit the ability to control meeting room temperatures. It's a tradeoff.

Specifically in reply to your question:
"what percentage of their air intake is outside air, and is it possible to increase that to 100%." 

I received the following from the hotel:
"Percentage of outside air is variable due to a number of factors such as temperature and exterior air quality (eg forest fires), and is anywhere between 10% and 90%."

Susan

**Comment by danbri** [comment]
> Maybe it would be worthwhile to have a co2 air quality monitor on site?

Sorry - only just saw this discussion. I very strongly support @gosko ’s suggestion here.

**Comment by gosko** [comment]
I picked up an Aranet4 co2 monitor the other day and will have it with me at TPAC.
